ui/app/controllers/vault/cluster/access/oidc/clients/client.js

```diff
@@ -5,22 +5,7 @@
 
 import Controller from '@ember/controller';
 import { service } from '@ember/service';
-import { tracked } from '@glimmer/tracking';
 
 export default class OidcClientController extends Controller {
   @service router;
-  @tracked isEditRoute;
```

Comment:
This was causing a strange error in login tests? The constructor was being called in an unrelated, oidc auth login test. It seemed like the controller was being used when it shouldn't have been. So I refactored to a better pattern of just checking the route in the template which is how I refactored this.